dusk-labyrinth.lic

```diff
@@ -98,9 +97,9 @@ end
 before_dying do
   Flags.delete('five-minute-warning')
   Flags.delete('done')
-  Flags.delete('no-monkey')
-  Flags.delete('monkey')
+  Flags.delete('no-pet')
+  Flags.delete('pet')
   Flags.delete('caught')
 end
 
-DuskLab.new
+DuskLab.new(get_settings)
```

Comment:
Super minor, not neccessary to change, but I prefer to have `get_settings` called inside `initialize`. That is consistent with other scripts and makes it easier to change in the future if we need to pass args to `get_settings`.

Reply:
Happy to make the change for consistency. Fixed per https://github.com/rpherbig/dr-scripts/pull/4390/commits/8a4cf39c89ef5ef91b928f5c7a0b48ee09652b68 and https://github.com/rpherbig/dr-scripts/pull/4390/commits/bbc737659e907915bb235f6778da1a248ab94456